### PR TITLE
Kibana reports troubleshooting

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/kibana/reporting.md [hunk 1]
```diff
@@ -39,3 +39,13 @@ Definitions let you schedule reports for periodic creation.
    For scheduled reports, choose either **Recurring** or **Cron based**. You can receive reports daily or at some other time interval. Cron expressions give you even more flexiblity. See [Cron expression reference](../../alerting/cron/) for more information.
 
 1. Choose **Create**.
+
+## Troubleshooting
```
Comment (aetter) [inline docs/kibana/reporting.md:43]: Hey @ashwinkumar12345, this looks like more of an installation issue than an issue you'd encounter when using the plugin. Would a user encounter this when they attempt to start Kibana (`./bin/kibana`)? If so, should we name this section "Chromium fails to launch with Kibana" and give the condition under which they'd encounter it (plus a formal error string?).

File: docs/kibana/reporting.md [hunk 2]
```diff
@@ -39,3 +39,13 @@ Definitions let you schedule reports for periodic creation.
...
+
+### Chromium fails to launch
+
+This problem occurs due to two reasons:
+
+1. You don't have the correct version of `headless-chrome` to match the OS on which Kibana is running. Download the correct version of `headless-chrome` from [here](https://github.com/opendistro-for-elasticsearch/kibana-reports/releases/tag/chromium-1.12.0.0).
+
+2. You're missing additional dependencies. Install the required dependencies for your OS from the [additional libraries](https://github.com/opendistro-for-elasticsearch/kibana-reports/tree/dev/kibana-reports/rendering-engine/headless-chrome) section.
```
Comment (zhongnansu) [inline docs/kibana/reporting.md:51]: Np: should this link include the title anchor? https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/dev/kibana-reports/rendering-engine/headless-chrome/README.md#additional-libaries

File: docs/kibana/reporting.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/kibana/reporting.md:43]: It's when the user tries to generate a report and sees a download error. I added the error message.

Comment (aetter) [reply docs/kibana/reporting.md:43]: :thumbsup:

**Pull request merged**